app/assets/javascripts/discourse/app/components/summary-box.hbs

```diff
@@ -0,0 +1,62 @@
+{{#if @postAttrs.hasTopRepliesSummary}}
+  <p>{{html-safe this.topRepliesSummaryInfo}}</p>
+{{/if}}
+<div class="summarization-buttons">
+  {{#if @postAttrs.summarizable}}
+    {{#if this.canCollapseSummary}}
+      <DButton
+        @class="btn btn-primary topic-strategy-summarization"
+        @action={{this.toggleSummary}}
+        @title="summary.buttons.hide"
+        @label="summary.buttons.hide"
+        @icon="chevron-up"
+      />
+    {{else}}
+      <DButton
+        @class="btn btn-primary topic-strategy-summarization"
```

Comment:
```suggestion
        @class="btn-primary topic-strategy-summarization"
```
(already [included in d-button](https://github.com/discourse/discourse/blob/4d2d31e68aa9ba556c6aafac98ebe0158d42b52e/app/assets/javascripts/discourse/app/components/d-button.gjs#L27-L27))